src/goto-programs/resolve_concrete_function_call.cpp

```diff
@@ -0,0 +1,87 @@
+/*******************************************************************\
+
+ Module: Goto Program utilties
```

Comment:
Oh no, I thought you had copy pasted the header from a different file because the name didn't seem to correspond to the file name.